### PR TITLE
Cleanup samples output

### PR DESCRIPTION
**Pull request by ashbeitz**

### CODE REVIEW

File: samples/mixed/consumer/src/main.rs [hunk 1]
```diff
@@ -55,9 +61,9 @@ fn start_show_notification_repeater(provider_uri: String, consumer_uri: String)
                 Err(status) => warn!("{status:?}"),
             }
 
-            info!("Invoked the show-notification command on endpoint {provider_uri}");
+            debug!("Completed the invoke request");
 
-            sleep(Duration::from_secs(5)).await;
+            sleep(Duration::from_secs(55)).await;
```
Comment (wilyle) [inline samples/mixed/consumer/src/main.rs:66]: should these sleep durations be configurable?

File: samples/mixed/consumer/src/main.rs [hunk 2]
```diff
@@ -71,6 +77,8 @@ fn start_activate_air_conditioning_repeater(provider_uri: String) {
     tokio::spawn(async move {
         let mut is_active = true;
         loop {
+            info!("Sending a set request for entity id {} to the value '{is_active}' to provider URI {provider_uri}", sdv::vehicle::cabin::hvac::is_air_conditioning_active::ID);
```
Comment (wilyle) [inline samples/mixed/consumer/src/main.rs:80]: suggest a newline

File: samples/command/consumer/src/main.rs [hunk 3]
```diff
@@ -28,6 +31,10 @@ fn start_show_notification_repeater(provider_uri: String, consumer_uri: String)
     debug!("Starting the Consumer's show notification repeater.");
     tokio::spawn(async move {
         loop {
+            let payload: String = String::from("The show-notification request.");
+
+            info!("Sending an invoke request on entity {} with payload '{payload} to provider URI {provider_uri}", sdv::vehicle::cabin::infotainment::hmi::show_notification::ID,);
```
[the remark this answers is not in the document]
Comment (ashbeitz) [reply samples/command/consumer/src/main.rs]: I fixed the comma.  I will let cargo fmt decide when to wrap lines.

File: samples/command/consumer/src/main.rs [hunk 4]
```diff
@@ -127,7 +133,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let provider_uri = String::from(uri_str_option.unwrap());
     info!("The URI for the show-notification command's provider is {provider_uri}");
 
-    let consumer_uri = format!("http://{consumer_authority}"); // Devskim: ignore DS137138
+    let consumer_uri = format!("http://{CONSUMER_ADDR}"); // Devskim: ignore DS137138
```
[the remark this answers is not in the document]
Comment (ashbeitz) [reply samples/command/consumer/src/main.rs:136]: We do when it is accompanied by a "http://".

File: samples/command/consumer/src/main.rs [hunk 5]
```diff
@@ -70,21 +75,22 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
...
     let addr: SocketAddr = consumer_authority.parse()?;
     let consumer_impl = consumer_impl::ConsumerImpl::default();
     let server_future =
         Server::builder().add_service(ConsumerServer::new(consumer_impl)).serve(addr);
+    info!("The HTTP server is listening on address '{CONSUMER_ADDR}'");
 
     // Obtain the DTDL for the send_notification command.
-    info!("Sending a find_by_id request to the Digital Twin Service for the DTDL for the send_notification command.");
-    let mut client = DigitalTwinClient::connect("http://[::1]:50010").await?; // Devskim: ignore DS137138
+    info!("Sending a find_by_id request for entity id {} to the In-Vehicle Digital Twin Service URI {IN_VEHICLE_DIGITAL_TWIN_SERVICE_URI}", sdv::vehicle::cabin::infotainment::hmi::show_notification::ID);
```
[the remark this answers is not in the document]
Comment (ashbeitz) [reply samples/command/consumer/src/main.rs:86]: I think that it is fine.

[the remark this answers is not in the document]
Comment (ashbeitz) [reply samples/command/consumer/src/main.rs:86]: Done

File: samples/mixed/consumer/src/main.rs (hunk 1 above)
Comment (ashbeitz) [reply samples/mixed/consumer/src/main.rs:66]: No

File: samples/mixed/consumer/src/main.rs (hunk 2 above)
Comment (ashbeitz) [reply samples/mixed/consumer/src/main.rs:80]: Fixed

File: samples/property/consumer/src/main.rs [hunk 6]
```diff
@@ -25,20 +29,20 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
...
-    let addr: SocketAddr = "[::1]:60010".parse()?;
+    let addr: SocketAddr = CONSUMER_ADDR.parse()?;
     let consumer_impl = consumer_impl::ConsumerImpl::default();
     let server_future =
         Server::builder().add_service(ConsumerServer::new(consumer_impl)).serve(addr);
 
     // Obtain the DTDL for the ambient air temmpterature.
-    info!("Sending a find_by_id request to the Digital Twin Service for the DTDL for ambient air temperature.");
-    let mut client = DigitalTwinClient::connect("http://[::1]:50010").await?; // Devskim: ignore DS137138
+    info!("Sending a find_by_id request for entity id {} to the In-Vehicle Digital Twin Service URI {IN_VEHICLE_DIGITAL_TWIN_SERVICE_URI}", sdv::vehicle::cabin::hvac::ambient_air_temperature::ID);
```
[the remark this answers is not in the document]
Comment (ashbeitz) [reply samples/property/consumer/src/main.rs:38]: Fixed,

File: samples/property/provider/src/main.rs [hunk 7]
```diff
@@ -46,7 +49,7 @@ fn start_ambient_air_temperature_data_stream(subscription_map: Arc<Mutex<Subscri
             }
 
             for url in urls {
-                info!("Publishing the ambient air temperature as {temperature} to {url}");
+                info!("Sending a publish request for {} with value {temperature} to consumer URI {url}", sdv::vehicle::cabin::hvac::ambient_air_temperature::ID);
```
[the remark this answers is not in the document]
Comment (ashbeitz) [reply samples/property/provider/src/main.rs]: Fixed

**Pull request merged**